### PR TITLE
Windows for Server

### PR DESCRIPTION
**Pull request by rosieyohannan**
(_3rd time I've made this PR - each time I rebase I end up with commits in the PR that aren't relevant. clearly doing something wrong!_)

All changes for Windows on Server

### CODE REVIEW

**Comment by appplemac** [comment]
I left some comments on the previous iteration of this PR, a bit hard to understand what’s new. I’ll assume that all my feedback from the previous PR has been incorporated.

The changes to the Hello World page LGTM.

File: jekyll/_cci2/vm-service.adoc [hunk 1]
```diff
@@ -66,6 +68,13 @@ You will need to associate the `circleci` user with the image you want to use as
 
 To enable the use of these custom images for your installation, you will need to provide the individual AMI IDs in the Settings accessible from the Management console (for example, `<your-circleci-id>.com:8800`).
 
+=== Creating a Windows AMI
+_Introduced in CircleCI Server v2.18.3_
+
+Creating a Windows image and specifying it under the VM Service settings lets your users run jobs on dedicated Windows VMs. To create your Windows image run through the steps listed in our [image builder repo](https://github.com/CircleCI-Public/circleci-server-windows-image-builder), then copy the generated AMI ID and paste into the Custom Windows VM AMI field in your Management Console settings, under VM Provider (for example, `<your-hostname.com:8800/settings>`).
```
Comment (rosieyohannan) [inline jekyll/_cci2/vm-service.adoc]:
```suggestion
Creating a Windows image and specifying it under the VM Service settings lets your users run jobs on dedicated Windows VMs. To create your Windows image run through the steps listed in our https://github.com/CircleCI-Public/circleci-server-windows-image-builder[image builder repo], then copy the generated AMI ID and paste into the Custom Windows VM AMI field in your Management Console settings, under VM Provider (for example, `<your-hostname.com:8800/settings>`).
```

File: jekyll/_cci2/vm-service.adoc [hunk 2]
```diff
@@ -66,6 +68,13 @@ You will need to associate the `circleci` user with the image you want to use as
 
 To enable the use of these custom images for your installation, you will need to provide the individual AMI IDs in the Settings accessible from the Management console (for example, `<your-circleci-id>.com:8800`).
 
+=== Creating a Windows AMI
+_Introduced in CircleCI Server v2.18.3_
+
+Creating a Windows image and specifying it under the VM Service settings lets your users run jobs on dedicated Windows VMs. To create your Windows image run through the steps listed in our https://github.com/CircleCI-Public/circleci-server-windows-image-builder[image builder repo], then copy the generated AMI ID and paste into the Custom Windows VM AMI field in your Management Console settings, under VM Provider (for example, `<your-hostname.com:8800/settings>`).
```
Comment (rosieyohannan) [inline jekyll/_cci2/vm-service.adoc]: @iynere here's where we send people to the windows image builder...

File: docs/server-docs-release-process.adoc [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+:page-liquid:
+:icons: font
+:toc: macro
+:toc-title:
+
+== List of jobs to complete before releasing
+
+* Update version under the `server_version` key in `jekyll/_config.yml`.
+* Make operational changes required for the release.
+* If anyth
```
Comment (teesloane) [inline docs/server-docs-release-process.adoc]: What is this?

File: jekyll/_cci2/v.2.18-overview.md [hunk 4]
```diff
@@ -20,6 +20,10 @@ This document provides a summary of features and product notes for the release o
 * We have made some changes to our Redis configuration. If you have externalized Redis then you’ll need to update your configuration. Please contact your Customer Success Manager.
 * We have also made changes to our Postgres version and require at least postgreSQL v9.5.16. If you have externalized postgreSQL then please update to at least that version in 2.17.x before upgrading to 2.18.
 
+## What's New in Release 2.18.3
+
+* You can now generate a Windows-based image to make available for running using the `machine` executor. See our VM Service guide for [instructions on building the image and making it available](https://circleci.com/docs/2.0/vm-service/#creating-a-windows-ami). For details of using the Windows executor, see our [Executors and Images overview](https://circleci.com/docs/2.0/executor-intro/#windows).
```
Comment (teesloane) [inline jekyll/_cci2/v.2.18-overview.md]: I'd recommend using the `{{site.baseurl}}` for internal links.

File: docs/server-docs-release-process.adoc (hunk 3 above)
Comment (rosieyohannan) [reply docs/server-docs-release-process.adoc]: 😭good question, will fix! - this is mainly a list for me so I don't miss anything. Must have got distracted :-O.

**Pull request merged**